tests/test_typing.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+
+from ppb_vector import Vector2
+
+# List of operations that (Vector2, Vector2) -> Vector2
```

Comment:
This suggest that the variable names could be more clear. It took me a few moments to realize the name meant "Binary operations."